src/components/scores/concept-coach-cell.cjsx

```diff
@@ -10,15 +10,49 @@ ConceptCoachCell = React.createClass
 
   mixins: [CellStatusMixin] # prop validation
 
+  getInitialState: ->
+    placement: 'right'
+
+  setPopoverPosition: ->
```

Comment:
@openstax/tutor-fe : Would this be useful in other popovers? I recall places like the roster moving the position over to the left because the links were on the right side of the page and the link hover popups doing something similar